samples/kubernetes/end2end/docs/existing-domain.md

```diff
@@ -0,0 +1,36 @@
+## Monitoring an Existing WebLogic Domain
+In this step, we show you how to integrate Prometheus and Grafana with an existing WebLogic domain.
+
+Assume you have already followed [Manage Domains](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/) guide in the WebLogic Kubernetes Operator to run a WebLogic domain and have followed previous steps in this sample to setup Prometheus and Grafana.
+
+### Installing the wls-exporter to the WebLogic domain
+You need to install the wls-exporter web application to the WebLogic domain.
+
+First, generate the wls-exporter war file following the [building from source](https://github.com/oracle/weblogic-monitoring-exporter#building-from-source) guide using the configuration file in this sample [exporter-config.yaml](../dashboard/exporter-config.yaml). Then deploy the war to the WebLogic servers and clusters using WLST or WDT.  
+
+After the wls-exporter is installed successfully, you can check the exported WebLogic Runtime Metrics from each server, see the detail guide [here](04-wls-domain.md#check-the-weblogic-runtime-metrics).
+
+### Updating Prometheus Configuration
+Next we need to inform Prometheus server to scrape metric from the WebLogic domain. Basically we need to add a new job to prometheus.yml, the configuration file of Prometheus server. The job is similar to what we added for `domain1` in [values.yaml](../prometheus/values.yaml#L59) of Prometheus chart. As shown below, lines in red need to be updated properly.
```

Comment:
Next we need to inform Prometheus server to scrape metric from the WebLogic domain. -> Next, we need to inform the Prometheus server to scrape metrics from the WebLogic domain.
Basically we need to add a new job to prometheus.yml, the configuration file of Prometheus server. -> Basically, we need to add a new job to `prometheus.yml`, the configuration file of the Prometheus server.
of Prometheus chart. -> of the Prometheus chart.